FirebasePerformance/Sources/Loggers/FPRGDTLogger.m

```diff
@@ -54,7 +54,7 @@ - (instancetype)initWithLogSource:(NSInteger)logSource {
   return self;
 }
 
-- (void)logEvent:(FPRMSGPerfMetric *)event {
+- (void)logEvent:(struct _firebase_perf_v1_PerfMetric)event {
```

Comment:
Why do we need `_` in front of `_firebase_perf_v1_PerfMetric`. Can we use the struct definition that was generated?

Reply:
They are the same. I have changed all of them to `firebase_perf_v1_PerfMetric`.